Gemfile

```diff
@@ -21,7 +21,7 @@ gem 'parallel_split_test', git: 'https://github.com/animoto/parallel_split_test.
 gem 'allure-rspec', git: 'https://github.com/animoto/allure-rspec.git', branch: 'rspec-2'
 gem 'compass'
 gem 'zurb-foundation'
-gem 'logging'
+gem 'logging', '2.0.0'
```

Comment:
Hmmm, I'm probably at fault for this, but is there a reason why this is both in the Gemfile and Gemspec?